src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Color[] COLORS = Color.values();
```

Comment:
According to the checklist, don't use static methods or fields in your solution unless it is a constant. Here, `COLORS` is an array of enum constants, which can be considered as a constant, so it's fine to have it as a `static final` field.